src/ds/d500/d500-color.cpp

```diff
@@ -0,0 +1,176 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#include <cstddef>
+#include "metadata.h"
+
+#include "ds/ds-timestamp.h"
+#include "proc/color-formats-converter.h"
+#include "d500-options.h"
+#include "d500-color.h"
+
+namespace librealsense
+{
+    std::map<uint32_t, rs2_format> d500_color_fourcc_to_rs2_format = {
+         {rs_fourcc('Y','U','Y','2'), RS2_FORMAT_YUYV},
+         {rs_fourcc('Y','U','Y','V'), RS2_FORMAT_YUYV},
+         {rs_fourcc('U','Y','V','Y'), RS2_FORMAT_UYVY},
+         {rs_fourcc('M','J','P','G'), RS2_FORMAT_MJPEG},
+         {rs_fourcc('B','Y','R','2'), RS2_FORMAT_RAW16},
+         {rs_fourcc('M','4','2','0'), RS2_FORMAT_M420}
```

Comment:
@maloel do we have M420 on d500 public project?

Reply:
"Aye"?

Reply:
Yes :)
M420 on DDS, anyway. For USB, not a priority.
Noted; nothing to do.